_CoqProject

```diff
@@ -11,4 +11,6 @@ src/cvc4Solver.ml
 src/polyaSolver.ml
 src/smtTactic.mllib
 
+theories/Coqlib.v
 theories/Tactic.v
+theories/Integers.v
```

Comment:
This also seems like it should not belong in this PR. If we do this, then we are only supporting *this* `Integers` library which is not a good way to go.